libretroshare/src/retroshare/rsdisc.h

```diff
@@ -1,9 +1,10 @@
 /*******************************************************************************
- * libretroshare/src/retroshare: rsdht.h                                       *
+ * RetroShare remote peers gossip discovery                                    *
```

Comment:
normally we put the file path here. So it should be libretroshare/src/retroshare/rsdisc.h

Reply:
Many times the file path/name there is wrong in our code, also because it doesn't add any information, I do prefer to put a short message descriptive of the file there, but if the file name there is some Debian obnoxious policy I can change that to the path of the file again.